recipes/scippp/all/conanfile.py

```diff
@@ -34,9 +34,9 @@ def _min_cppstd(self):
     @property
     def _compilers_minimum_version(self):
         return {
-            "gcc": "7",
+            "gcc": "8",
```

Comment:
I'm curious why did you need to change the minimum compiler version? In case this new version is not compatible (e.g. required C++ standard), it should be validated by package version instead. Otherwise, users will lose access to older versions of this library.

Reply:
Indeed the old versions works fine with the older compiler. The new compiler is now only enforced for the new version.

Reply:
Thank you for checking it!